sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -942,6 +895,7 @@ def test_dict_learning_online_numerical_consistency(method):
     U_64, V_64 = dict_learning_online(
```

Comment:
I have this test failing for a single value locally and I am not sure why. I set a larger number of iteration since if it was passing before, we probably did `1_000` but it does not change anything.

Reply:
try ``tol=0`` and ``max_no_improvement=None`` to disable early stopping.

Reply:
Yep it works.